Pipfile.lock

```diff
@@ -66,10 +66,10 @@
         },
         "django-js-asset": {
             "hashes": [
-                "sha256:0dd2c5f64f2b24eb8a7270a6a59cb914a03f205335bd0eb6207bf61cf7410828",
-                "sha256:c6637e4f3fef12024564057a48bef484db6491c610273d76d25d8dbe6ecc4ac1"
+                "sha256:bcd00d7dc67f86cad1dac4bf7b43fa4b75088c214d2c92002b8705c6a7362e7e",
+                "sha256:38fc1de7a93a2f667fe7ee34acc53d682bac5ce6c73ac6f0ecb4ff77d0f61f1b"
             ],
-            "version": "==0.1.1"
+            "version": "==1.0.0"
```

Comment:
That's the problem of not adding constraints to the `Pipfile`. I would expect changes and BC breaks between `0.1.1` and `1.0.0`. Can we do something for production-related packages?

Reply:
I understand your concern. That's a second degree dependency. There is something I still don't understand in the `Pipfile.lock`'s generation/update: when installing a new dependency _via_: `pipenv install foo`, `pipenv` will automatically upgrade all dependencies. Most of the time you do not want that. Dependencies should be explicitly upgraded, _i.e._ when I ask for it.

Reply:
Yes, and there is an issue for that on Pipenv. Apparently, it wont get solved.. :(